CHANGELOG-draft.md

```diff
@@ -32,14 +32,15 @@ THIS FILE ACCUMULATES THE RELEASE NOTES FOR THE UPCOMING RELEASE.
 ## API Changes
 
 * Deprecate `DELETE /conversations/:cnv/members/:usr` (#1697)
-* Add `DELETE /conversations/:cnv/members/:domain/:usr` (#1697)
+* Add `DELETE /conversations/:domain/:cnv/members/:domain/:usr` (#1697)
```

Comment:
`domain` twice?  or is it:

```suggestion
* Add `DELETE /conversations/:domain/:cnv/members/:usr` (#1697)
```

Reply:
There really is a domain path component appearing twice: the first one qualifies a conversation, and the second a conversation member.